src/main/kotlin/io/hsar/wh40k/combatsimulator/model/UnitInstance.kt

```diff
@@ -20,12 +22,11 @@ class UnitInstance(
         val equipment: List<EquipmentItem>,
         val attackExecutor: AttackExecutor = AttackExecutor(), // open for test
         val startingAttributes: Map<Attribute, AttributeValue> =  // #TODO: Figure out whether this is good long-term solution
-                DEFAULT_ATTRIBUTES + equipment.map { it.modifiesAttributes }.sum(),
+                DEFAULT_ATTRIBUTES + equipment.map { it.modifiesAttributes }.sum()
+                        + Pair(Attribute.CURRENT_HEALTH, NumericValue(unit.stats.baseStats.getValue(BaseStat.MAX_HEALTH))),
```

Comment:
Common syntactic sugar for creating `Pair("foo", "bar")` is `"foo" to "bar"`.

Reply:
Updated